interface/code_systems/dsmiv_howto.php

```diff
@@ -15,7 +15,7 @@
  * You should have received a copy of the GNU General Public License
```

Comment:
Delete this file.  It is based on the bad assumption that DSM4 codes actually exist.  It is DSM5 now anyway, and it only refers to ICD10...DSM is a guide, not a code set.